shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/java/org/apache/shardingsphere/proxy/Bootstrap.java

```
@@ -157,6 +162,26 @@ private static void initialize(final Authentication authentication, final Proper
         log(authentication, properties);
         initControlPanelFacade(metricsConfiguration);
         initCluster(cluster);
+        updateServerInfo();
+    }
+    
+    private static void updateServerInfo() {
+        List<String> schemaNames = ProxySchemaContexts.getInstance().getSchemaNames();
+        if (CollectionUtils.isEmpty(schemaNames)) {
+            return;
+        }
+        Map<String, DataSource> dataSources = ProxySchemaContexts.getInstance().getSchema(schemaNames.get(0)).getSchema().getDataSources();
+        DataSource singleDataSource = dataSources.values().iterator().next();
+        try (Connection connection = singleDataSource.getConnection()) {
+            DatabaseMetaData meta = connection.getMetaData();
+            String name = meta.getDatabaseProductName();
+            String version = meta.getDatabaseProductVersion();
+            log.info("server name {} , server version {}", name, version);
+            MySQLServerInfo.setServerVersion(version);
+        } catch (SQLException e) {
+            log.error(e.getMessage(), e);
```

throw a ShardingSphereException instead of print log.